hardhat.config.js

```diff
@@ -0,0 +1,15 @@
+require("@nomiclabs/hardhat-truffle5");
+
+module.exports = {
+  networks: {
```

Comment:
How do we deploy now? You don't need to put public testnet specific info which you had on `truffle.js` for Ropsten

Reply:
I'll need to write a new deployment script when the time comes.